proposals/0461-async-function-isolation.md

```diff
@@ -337,7 +337,7 @@ struct S: Sendable {
 ```
 
 Only (implicitly or explicitly) `nonisolated` functions can be marked with
-`@concurrent`; it is an error to use the these attributes with
+`@concurrent`; it is an error to use these attributes with
```

Comment:
Thanks! I think there's only one attribute being referred to here (namely, `@concurrent`).

```suggestion
`@concurrent`; it is an error to use this attribute with
```